lucille-plugins/lucille-tika/src/main/java/com/kmwllc/lucille/tika/stage/TextExtractor.java

```diff
@@ -136,6 +136,15 @@ public void parseInputStream(Document doc, InputStream inputStream) {
       parser.parse(inputStream, bch, metadata, parseCtx);
     } catch (IOException | SAXException | TikaException e) {
       log.warn("Tika Exception: {}", e.getMessage());
+    } finally {
+      // close the inputStream regardless if error is thrown
+      if (inputStream != null) {
+        try {
+          inputStream.close();
```

Comment:
pls. update the javadoc for this method (l130 above) to say that it closes the inputStream